src/cli.js

```diff
@@ -98,8 +99,9 @@ commander
 commander
   .command("build")
   .description("create production asset build")
+  .option(...prepareStaticOption)
   .action(checkGluestickProject)
-  .action(() => build())
+  .action((options) => build(options.static))
```

Comment:
just adding the option listed above so you can pass `--static` to the build command. The value from that will be true or false and passed to the `build` command in src/commands/build.js